test/NuGet.Tests.Apex/README.md

```diff
@@ -0,0 +1,73 @@
+# NuGet.Client Apex Tests
+
+This is a collection of end to end Visual Studio tests built on top of the Apex framework.
+All the tests cases are contained within the [NuGet.Tests.Apex](../NuGet.Tests.Apex/NuGet.Tests.Apex.csproj) project.
```

Comment:
Well, except OptProf, although I guess techinically that's for training, not testing 😆

Reply:
Yep, fair :D